mitiq/pec/types/tests/test_types.py

```diff
@@ -27,81 +28,61 @@
     OperationRepresentation,
 )
 
+icirq = Circuit(cirq.I(cirq.LineQubit(0)))
+xcirq = Circuit(cirq.X(cirq.LineQubit(0)))
+ycirq = Circuit(cirq.Y(cirq.LineQubit(0)))
+zcirq = Circuit(cirq.Z(cirq.LineQubit(0)))
+hcirq = Circuit(cirq.H(cirq.LineQubit(0)))
+cnotcirq = Circuit(cirq.CNOT(cirq.LineQubit(0), cirq.LineQubit(1)))
 
-def test_init_with_gate():
-    ideal_gate = cirq.Z
+
+def test_init_with_cirq_circuit():
     real = np.zeros(shape=(4, 4))
-    noisy_op = NoisyOperation.from_cirq(ideal_gate, real)
+    noisy_op = NoisyOperation(zcirq, real)
     assert isinstance(noisy_op._circuit, cirq.Circuit)
 
-    assert _equal(
```

Comment:
Not quite sure why this statement was removed, can you clarify?

Reply:
Ah, okay I see it now. I think it the multi-line formatting threw me off, all the better to simply it.